src/trio/_subprocess.py

```diff
@@ -766,6 +766,7 @@ async def killer() -> None:
 
                 nursery.start_soon(killer)
                 await proc.wait()
+                # TODO: source/reason?
                 killer_cscope.cancel()
```

Comment:
I think I'd just leave this one alone; it's very much an implementation detail and will be caught by the shielded scope before any user sees it.